xml/System.Reflection.Metadata.Ecma335/BlobEncoder.xml

```diff
@@ -101,9 +101,9 @@
         <Parameter Name="namedArguments" Type="System.Action&lt;System.Reflection.Metadata.Ecma335.CustomAttributeNamedArgumentsEncoder&gt;" Index="1" FrameworkAlternate="netcore-1.1;netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="fixedArguments">To be added.</param>
-        <param name="namedArguments">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="fixedArguments">Called first, to encode fixed arguments.</param>
+        <param name="namedArguments">Called second, to encode named arguments.</param>
+        <summary>Encodes Custom Attribute Signature blob.</summary>
```

Comment:
capitalization:

```suggestion
        <summary>Encodes custom attribute signature blob.</summary>
```

Reply:
Thanks. I feel like this also needs to disambiguate with the rest of the overloads. How about this?:

```suggestion
        <summary>Encodes the custom attribute signature blob using the specified argument encoding actions.</summary>
```